signing_clients/apps.py

```diff
@@ -319,7 +319,7 @@ def make_signed(self, signature, outpath=None, sigpath=None):
         if os.path.exists(outpath):
             raise IOError("File already exists: %s" % outpath)
 
-        sigpath = sigpath or signature.filename
+        sigpath = sigpath or self.filename
```

Comment:
right I was confused about the filename in the object, does your test catch this case?